src/style.css

```diff
@@ -1 +1,34 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  height: 20px;
+}
+
+.stars:last-of-type {
+  height: 16px;
+}
```

Comment:
Redundant, because you have height here 
<img width="180" alt="image" src="https://github.com/user-attachments/assets/d5bae374-1185-4da0-8f05-6885ffeb394a">
```suggestion
```

Reply:
Hi @volodymyr-soltys97 
.stars  & .stars__star   are not the same

.stars  height used for the vertical gap between rows
.stars__star heigth used for the background position